### DISCUSSION TITLE
Change Detection not enabled for hybrid/dual model?

### DISCUSSION THREAD

**Discussion by hayescode** [TabularEditor/TabularEditor3#1137] [🙋♀️TE3 Q&A]
**Describe the issue**
I have a model with 1 hybrid fact table and 2 dual dimension tables. When connecting my report to this model the page refresh option allows for page refresh at a set interval, but change detection is not available.

Setting this exact scenario up in Power BI Desktop (start with import and create RangeStart/RangeEnd) keeps change detection available.

** Which version are you currently using? **
Tabular Editor 3.6.0

**To Reproduce**
My TE3 model is published to the Power BI service and I am connecting to it locally using Power BI Desktop.

**Expected behavior**
I would expect change detection to be enabled.

**Screenshots**
![image](https://github.com/TabularEditor/TabularEditor3/assets/35790761/3be33da9-c4cf-47fd-940e-9d9c9332030b)

**Desktop (please complete the following information):**
Windows 10

**Additional context**

**Comment by DBojsen** [comment]
Hi @hayescode 

According to the [Microsoft documentation, this is only available when working on DirectQuery sources](https://learn.microsoft.com/en-us/power-bi/create-reports/desktop-automatic-page-refresh#authoring-reports-with-automatic-page-refresh-in-power-bi-desktop):

It is not something that we can enable and disable as an external tool.

BR David - TE3 support

**Comment by hayescode** [comment]
@DBojsen i tried to follow the [documentation](https://docs.tabulareditor.com/te3/tutorials/incremental-refresh/incremental-refresh-modify.html#configure-detect-data-changes) with just a single DQ table. The connected thin report cannot enable change detection in PBI Desktop, however you can enable in the PBI Service ([details](https://community.fabric.microsoft.com/t5/Service/change-detection-in-live-connection-models/m-p/1347685/highlight/true#M107791)). So the report can handle this, but I cannot configure change detection in TE. Therefore the data is not actually updating in the service/report.

How do you set this up the data model in TE to allow change detection for this scenario where i have just 1 DQ table.
